src/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,28 @@
+import LoginPagePhoto from '../../assets/LoginPagePhoto.svg'
+import CircleLogo from '../../assets/stamp.svg'
+import './LoginPage.css'
+
+export default function LoginPage() {
+  return (
+    <div className='wrap'>
+      <div className='imgHolder'>
+        <img src={CircleLogo} className='imgLogo' />
```

Comment:
img elements must have an alt prop, either with meaningful text, or an empty string for decorative images